src/main/java/org/sunbird/trending/controller/TrendingController.java

```diff
@@ -21,12 +22,12 @@ public ResponseEntity<Map<String, Object>> trendingSearch(
         return new ResponseEntity<>(compositeSearchRes, HttpStatus.OK);
     }
 
-    @PostMapping("admin/trending/content/search")
-    public ResponseEntity<Map<String, Object>> trendingContentSearch(
+    @PostMapping("/microsite/topFeatured/content/search")
+    public ResponseEntity<SBApiResponse> trendingContentSearch(
```

Comment:
Instead of using trendingContentSearch, derive a more generic name following the discussion with Sahil.